api/app/tests/fba/c3_response.json

```diff
@@ -19,13 +19,13 @@
             "build_up_index": 117.899,
             "duff_moisture_code": 103.923,
             "fire_weather_index": 35.947,
-            "head_fire_intensity": 8880.691840628606,
+            "head_fire_intensity": 8879.275214638277,
             "rate_of_spread": 7.0102999937431365,
             "fire_type": "IC",
-            "percentage_crown_fraction_burned": 0.6078811483509187,
-            "flame_length": 5.440800137427278,
-            "sixty_minute_fire_size": 7.12167902519349,
-            "thirty_minute_fire_size": 1.0808419414685175,
+            "percentage_crown_fraction_burned": 0.607295415601325,
+            "flame_length": 5.440366168631874,
+            "sixty_minute_fire_size": 7.1167278346877,
+            "thirty_minute_fire_size": 1.0795545716384733,
```

Comment:
just curious - what's the reason why some of these values have changed very slightly?

Reply:
Date of minimum foliar moisture content was set to 0, now using value from REDapp for C3 - 189